<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_window</class>
 <widget class="QMainWindow" name="Register_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>619</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>619</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Register_window{
	background-color: rgb(238, 238, 236);
	border-radius: 2px;
}

#line_name{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
	margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_name:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#line_last{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
	margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_last:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#line_phone{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
	margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_phone:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#line_email{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
	margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_email:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#line_pass{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
	margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_pass:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#line_user{	
	
	background-color: transparent;	
	border:none;
	color:rgb(46, 52, 54);
	border-bottom: 1px solid rgb(85, 87, 83);
margin-bottom: 20px;
	margin-left:10px;
	margin-right:10px;
}

#line_user:focus{
	
	border-bottom: 3px solid rgb(138, 226, 52);
}

#register_btn{
	background-image: url(:/icons/icons/degradado.jpg);
	border:1px solid rgb(211, 215, 207);
	border-radius:20px;
	font: 15pt &quot;DejaVu Serif&quot;;
	color:rgb(255, 255, 255);

}

#register_btn:hover{
	border:3px solid rgb(252, 233, 79);
	
	color: rgb(252, 233, 79);
}


#label_title{
font: 25pt &quot;DejaVu Serif&quot;;
color:rgb(0, 0, 0);
}

#label_name{

	background-image: url(:/icons/icons/name.png);
}

#label_last{

	background-image: url(:/icons/icons/name.png);
}

#label_phone{
	background-image: url(:/icons/icons/phone.png);

}

#label_email{

	background-image: url(:/icons/icons/email.png);
}


#label_user{
	background-image: url(:/icons/icons/user.png);
}

#label_pass{
	background-image: url(:/icons/icons/pass.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>531</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="line_name">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_phone"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="line_last"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_email"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="line_user"/>
     </item>
     <item>
      <widget class="QLineEdit" name="line_pass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_last">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pass">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
